grammars/clojure.cson

```diff
@@ -279,25 +279,24 @@
     'patterns': [
       {
         # ns, declare and everything that starts with def* or namespace/def*
-        'begin': '(?<=\\()(ns|declare|def[\\w\\d._:+=><!?*-]*|[\\w._:+=><!?*-][\\w\\d._:+=><!?*-]*/def[\\w\\d._:+=><!?*-]*)\\s+'
+        'begin': '(?<=\\()(ns|declare|def[0-9#\'A-Z_a-z!$%&+\\-.:<=>?|*]*|[A-Z_a-z!$%&+\\-.:<=>?|*][0-9#\'A-Z_a-z!$%&+\\-.:<=>?|*]*/def[0-9#\'A-Z_a-z!$%&+\\-.:<=>?|*]*)\\s+'
         'beginCaptures':
           '1':
             'name': 'keyword.control.clojure'
         'end': '(?=\\))'
-        'name': 'meta.definition.global.clojure'
         'patterns': [
           {
             # there may be some metadata before an actual definition
             'include': '#metadata'
           }
-          { # dynamic variables are rendered diferently
-            'include': '#dynamic-variables'
-          }
           {
             # recognizing a symbol as being defined here
             # copied and pasted from #symbol, screw it
-            'match': '([\\w\\.\\-\\_\\:\\+\\=\\>\\<\\!\\?\\*][\\w\\.\\-\\_\\:\\+\\=\\>\\<\\!\\?\\*\\d]+)'
-            'name': 'entity.global.clojure'
+            'match': '([A-Z_a-z!$%&+\\-.:<=>?|*][0-9#\'A-Z_a-z!$%&+\\-.:<=>?|*]*)'
+            'name': 'meta.definition.global.clojure entity.name.global.clojure'
```

Comment:
Scopes != CSS classes and while this may look correct in the DOM it will appear as one scope internally.